region/maridia/inner-pink.json

```diff
@@ -3717,21 +3675,12 @@
                 {
                   "name": "Halfie Climb Room Xray Climb Grapple Clip",
                   "notable": true,
+                  "entranceCondition": {
+                    "comeInWithDoorStuckSetup": {}
```

Comment:
There's probably an issue here, being on a node lock.
It should start on node 2 to clip into node1 door, however that should be written.

Reply:
I was thinking we could add a `bypassesLock` boolean as a strat property, to be able to represent strats like this (and then deprecate the `bypassStrats` lock property). If that sounds like a good solution, that could be added now. Otherwise, I guess postpone for later and just leave the strat where it was, inside the `bypassStrats` in the lock.

Reply:
It does fit the structure better if this strat were moved to links.  
Bypassing a door lock then makes the most sense as a strat property, yeah.  
IDK if there are other lock type bypasses, but you could just say it means bypassing door locks.  Like Draygon you can clip through the left side door with grapple and some other items, but grapple would let you do the fight too.